sopel/plugin.py

```diff
@@ -1112,10 +1112,27 @@ def rate(
         @rate(10, 10, 10, message='Hit the rate limit for this function.')
             # will send a NOTICE
 
-    The message can contain a placeholder for ``nick``: the message will be
-    formatted with the nick that hit rate limit::
-
-        @rate(10, 10, 2, message='Sorry {nick}, you hit the rate limit!')
+    The message can contain placeholders which will be filled in:
+
+        - ``nick``: the nick that hit the rate limit
+        - ``channel``: the channel in which the rate limit was hit (will be
+          ``'private-message'`` for private messages)
+        - ``sender``: the sender (nick or channel) of the message which hit
+          the rate limit
+        - ``plugin``: the name of the plugin that hit the rate limit
+        - ``label``: the label of the plugin handler that hit the rate limit
+        - ``time_left``: the time remaining before the rate limit expires, as
+          a string
+        - ``time_left_sec``: the time remaining before the rate limit expires,
+          expressed in number of seconds
+        - ``rate_limit``: the rate limit, as a string
+        - ``rate_limit_sec``: the rate limit, expressed in number of seconds
+        - ``rate_limit_type``: the type of rate limit that was hit (one of
+          ``user, group, global``)
```

Comment:
Restructured text doesn't use tab/space before a list item, this may even confused it. And you'll have to change the `-` by a `*`: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks